its/src/test/java/com/sonar/it/csharp/UnitTestProjectTypeProbingTest.java

```diff
@@ -1,94 +1,101 @@
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this program; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- */
-package com.sonar.it.csharp;
-
-import com.sonar.it.shared.TestUtils;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
-import org.sonarqube.ws.Issues;
-
-import java.io.IOException;
-
-import static com.sonar.it.csharp.Tests.ORCHESTRATOR;
-import static com.sonar.it.csharp.Tests.getIssues;
-import static org.assertj.core.api.Assertions.assertThat;
-
-public class UnitTestProjectTypeProbingTest {
-  private static final String PROJECT = "UTProjectProbing";
-  private static final String MAIN_RULE_ID = "csharpsquid:S1048";
-  private static final String MAIN_AND_TEST_RULE_ID = "csharpsquid:S101";
-  private static Boolean isProjectAnalyzed = false;
-
-  @Rule
-  public TemporaryFolder temp = TestUtils.createTempFolder();
-
-  @Before
-  public void init() throws IOException {
-    if (!isProjectAnalyzed) {
-      TestUtils.reset(ORCHESTRATOR);
-      Tests.analyzeProject(temp, PROJECT, null);
-
-      isProjectAnalyzed = true;
-    }
-  }
-
-  @Test
-  public void mainProject_IsIdentifiedAsMain() {
-    assertThat(getIssues("UTProjectProbing:UTProjectProbing.Main/calculator.cs"))
-      .extracting(Issues.Issue::getRule)
-      .containsExactlyInAnyOrder(MAIN_RULE_ID, MAIN_AND_TEST_RULE_ID);
-  }
-
-  @Test
-  public void mainProject_WithPropertySetToTrue_IsIdentifiedAsTestProject() {
-    assertThat(getIssues("UTProjectProbing:UTProjectProbing.MainWithProjectPropertyTrue/calculator.cs")).isEmpty();
-  }
-
-  @Test
-  public void testProject_WithPropertySetToFalse_IsIdentifiedAsMainProjectByScannerAndTestByAnalyzer() {
-    assertThat(getIssues("UTProjectProbing:UTProjectProbing.MsTestWithProjectPropertyFalse/calculator.cs"))
-      .extracting(Issues.Issue::getRule)
-      .containsExactly(MAIN_AND_TEST_RULE_ID);
-  }
-
-  @Test
-  public void msTestProject_IsIdentifiedAsTestProject() {
-    assertThat(getIssues("UTProjectProbing:UTProjectProbing.MsTest/calculator.cs")).isEmpty();
-  }
-
-  @Test
-  public void xUnitProject_IsIdentifiedAsTestProject() {
-    assertThat(getIssues("UTProjectProbing:UTProjectProbing.xUnit/calculator.cs")).isEmpty();
-  }
-
-  @Test
-  public void project_WithTestInName_IsIdentifiedAsMainProject() {
-    assertThat(getIssues("UTProjectProbing:UTProjectProbing.ContainsTestInName/calculator.cs"))
-      .extracting(Issues.Issue::getRule)
-      .containsExactlyInAnyOrder(MAIN_RULE_ID, MAIN_AND_TEST_RULE_ID);
-  }
-
-  @Test
-  public void project_WithTestsSuffix_IsIdentifiedAsTestProject() {
-    assertThat(getIssues("UTProjectProbing:UTProjectProbing.EndsWithTests/calculator.cs")).isEmpty();
-  }
-}
+/*
```

Comment:
I'm not sure what happened here, it's IntelliJ that might have corrected the formatting....

Reply:
EOL?

Reply:
probably